hazelcast-client-new/src/test/java/com/hazelcast/client/map/ClientMapTest.java

```diff
@@ -226,6 +226,20 @@ public void testGetAllPutAll() {
     }
 
     @Test
+    public void testPutAllWithTooManyEntries() {
```

Comment:
See earlier comments. The point of the test was largely to exercise the block that used to form one branch, and wasn't to check if the operations were being sent to the correct partitions.